### ISSUE TITLE
Compute edges for polyhedra

### ISSUE THREAD

**Issue by paxperscientiam** [badzong/meshspin.js#9]
Edges should be computed rather than supplied by user, though could be toggleable.

**Comment by badzong** [comment]
This would be a nice feature for convex objects. However for concave objects it seems impossible because there are different edge combinations for the same node points.

**Comment by paxperscientiam** [comment]
Yeah, I looked into it some after making this post ... not trivial!

EDIT: I was recently thinking that there must exist computed tables of coordinates for various polyhedra. If found, it could then just a matter of reading it in.
